string command
Array position
---
string response